parcels/particlefile.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
```

Comment:
Yes and no. Conceptually yes, but since the internal format for grid and particle data is completely different, I'd say no for now. At the moment, all they have in common is the `import netcDF` statement, until we use a slightly higher-level abstraction that encapsulates various netCDF conventions , for example.